src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--30">
+      <div class="calendar_block cell calendar_block--start-day--sat"></div>
```

Comment:
Check correct modifiers for the calendar: you should set modifier for start day and for month length